### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#166] [Esercizi - Viaggi/Piazza]
![Es 1](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7f3ce4ec-309e-48b8-879e-a9d741a2ef4d)

**Comment by Elia-Belli** [comment]
- **1.1)** Siano $e,e'\in G, e\neq e'$ per assurdo, elementi neutri per $\cdot$ e $g \in G$:
 per definizione di elemento neutro $g\cdot e = g = e \cdot g$ e $g\cdot e' = g = e' \cdot g$
 siccome $e \in G$ possiamo sostituire $g$ con $e$: $e \cdot e' = e = e' \cdot e$, analogamente per $e' \in G$: $e' \cdot e = e' = e \cdot e'$
 ottenendo dalle uguaglianze $e' \cdot e = e \cdot e' \rightarrow  e = e'$ (contraddizione) $\Rightarrow$ l'elemento neutro è unico.
- **1.2)** Identico a #16 
- **1.3)** Identico a #17

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli

**Comment by daromacarlo** [comment]
[-Nota-18-18_annotato.pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13565806/-Nota-18-18_annotato.pdf)
